### PR TITLE
Fixed minor typos

### PR DESCRIPTION
**Pull request by APMichael**
- Fixed minor typos.

### CODE REVIEW

File: SandboxiePlus/SandMan/Windows/SettingsWindow.cpp [hunk 1]
```diff
@@ -1264,7 +1264,7 @@ void CSettingsWindow::UpdateCert()
 		ui.lblCert->setText(tr("<b>You have used %1/%2 evaluation certificates. No more free certificates can be generated.</b>").arg(EvalCount).arg(EVAL_MAX));
 	else
 		ui.lblCert->setText(tr("<b><a href=\"_\">Get a free evaluation certificate</a> and enjoy all premium features for %1 days.</b>").arg(EVAL_DAYS));
-	ui.lblCert->setToolTip(tr("You can request a free %1-day evaluation certificate up to %2 times for any one Hardware ID").arg(EVAL_DAYS).arg(EVAL_MAX));
+	ui.lblCert->setToolTip(tr("You can request a free %1-day evaluation certificate up to %2 times for any one hardware ID.").arg(EVAL_DAYS).arg(EVAL_MAX));
```
Comment (isaak654) [inline SandboxiePlus/SandMan/Windows/SettingsWindow.cpp:1267]:
```suggestion
	ui.lblCert->setToolTip(tr("You can request a free %1-day evaluation certificate up to %2 times per hardware ID.").arg(EVAL_DAYS).arg(EVAL_MAX));
```

File: SandboxiePlus/SandMan/Wizards/SetupWizard.cpp [hunk 2]
```diff
@@ -287,7 +287,7 @@ CCertificatePage::CCertificatePage(int iOldLevel, QWidget *parent)
     registerField("isEvaluate", m_pEvaluate);
 
     QLabel* pGetEvalCert = new QLabel(tr("<b><a href=\"_\"><font color='red'>Get a free evaluation certificate</font></a> and enjoy all premium features for %1 days.</b>").arg(EVAL_DAYS));
-    pGetEvalCert->setToolTip(tr("You can request a free %1-day evaluation certificate up to %2 times for any one Hardware ID").arg(EVAL_DAYS).arg(EVAL_MAX));
+    pGetEvalCert->setToolTip(tr("You can request a free %1-day evaluation certificate up to %2 times for any one hardware ID.").arg(EVAL_DAYS).arg(EVAL_MAX));
```
Comment (isaak654) [inline SandboxiePlus/SandMan/Wizards/SetupWizard.cpp:290]:
```suggestion
    pGetEvalCert->setToolTip(tr("You can request a free %1-day evaluation certificate up to %2 times per hardware ID.").arg(EVAL_DAYS).arg(EVAL_MAX));
```

**Comment by APMichael** [comment]
@isaak654 Thanks for the suggestion, but the pull request has already been merged.

**Comment by isaak654** [comment]
Perhaps next time.

**Comment by isaak654** [comment]
Perhaps next time?

**Comment by DavidXanatos** [comment]
You can make an other one I'll wait with the next build

**Comment by APMichael** [comment]
> You can make an other one I'll wait with the next build

Thank you for your reply. New pull request: #4182.

**Pull request merged**